projects/aducm_blinky_example/src.mk

```diff
@@ -0,0 +1,28 @@
+################################################################################
+#									       #
+#     Shared variables:							       #
+#	- PROJECT							       #
+#	- DRIVERS							       #
+#	- INCLUDE							       #
+#	- PLATFORM_DRIVERS						       #
+#	- NO-OS								       #
+#									       #
+################################################################################
+
+#In aducm projects the name of the source dir should be different from src
+#Direcory where app srcs are stored
+APP_SRCS = app_src
+
+SRC_FOLDERS +=	$(DRIVERS)/accel/adxl345
+#		$(NO-OS)/network 	 \
+		$(DRIVERS)/sd-card	 \
+		$(NO-OS)/libraries/fatfs
+
+#Include makefiles from each source directory if they exist
+DIRECORIES_WITH_MAKEFILES = $(wildcard $(foreach dir,$(SRC_FOLDERS),$(dir)\src.mk))
```

Comment:
Only the projects will have 'src.mk' files

Reply:
Explained in the next comment